nequip/data/AtomicData.py

```diff
@@ -24,6 +24,17 @@
 # A type representing ASE-style periodic boundary condtions, which can be partial (the tuple case)
 PBC = Union[bool, Tuple[bool, bool, bool]]
 
+_DEFAULT_SCALAR_FIELDS: Set[str] = {
+    AtomicDataDict.ATOMIC_NUMBERS_KEY,
+    AtomicDataDict.ATOM_TYPE_KEY,
+    AtomicDataDict.BATCH_KEY,
```

Comment:
?? these are all longs

Reply:
but we do want most `node_fields` to have the shape of `[n, 1]`, right? Please confirm.

Reply:
> but we do want most `node_fields` to have the shape of `[n, 1]`, right? Please confirm.

yes— it is more correct.

> this is the part that I'm not sure about. What's the dimension there? if I make them [n, 1], the model does not take it well.

Has nothing to do with their datatype?

But yes, I have to fix the shape logic